docs/userguides/testing.md

```diff
@@ -27,12 +27,10 @@ when it is the owner of the contract making the transaction.
 This is an example of how that test may look:
 
 ```python
-def test_is_owner(my_contract, owner, other):
+def test_is_owner(my_contract, owner, receiver):
     my_contract.set_owner(sender=owner)
     assert owner == my_contract.owner()
-
-    other_is_owner = my_contract.foo(sender=other)
-    assert not other_is_owner
+    assert not my_contract.is_owner(sender=receiver)
```

Comment:
I think this example would better if used a negative test.

```suggestion

    with ape.reverts("!authorized"):
        my_contract.authorized_method(sender=not_owner)
```

Reply:
Updated in dee36a56 including changing the previous paragraph wording